sklearn/tests/test_isotonic.py

```diff
@@ -27,6 +27,14 @@ def test_permutation_invariance():
     assert_array_equal(y_transformed, y_transformed_s)
 
 
+def test_check_increasing_small_number_of_samples():
+    x = [0, 1, 2]
+    y = [1, 1.1, 1.05]
+
+    is_increasing = assert_no_warnings(check_increasing, x, y)
+    assert_true(is_increasing)
```

Comment:
Looking at it a bit further I think you are right. From the name I assume `check_increasing` was checking that the `(x, y)` data points defined a function that was monotically increasing.